src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackServiceTest.java

```diff
@@ -314,54 +300,32 @@ void should_report_error_if_classification_is_supplementary_evidence() {
                 SUPPLEMENTARY_EVIDENCE.name()
             )
         );
-    }
-
-    @Test
-    void should_throw_InvalidCaseDataException_when_transformation_client_returns_422()
-        throws IOException, CaseTransformationException {
-        // given
-        when(s2sTokenGenerator.generate()).thenReturn(randomUUID().toString());
-        setUpTransformationUrl();
-        InvalidCaseDataException exception = new InvalidCaseDataException(
-            new HttpClientErrorException(HttpStatus.UNPROCESSABLE_ENTITY),
-            new TransformationErrorResponse(singletonList("error"), singletonList("warning"))
-        );
-        doThrow(exception)
-            .when(transformationClient)
-            .transformExceptionRecord(anyString(), any(ExceptionRecord.class), anyString());
 
-        // when
-        ProcessResult result = service.process(new CcdCallbackRequest(
-            EVENT_ID_CREATE_NEW_CASE,
-            caseDetails(basicCaseData()),
-            true
-        ), IDAM_TOKEN, USER_ID);
-
-        // then
-        assertThat(result.getExceptionRecordData()).isEmpty();
-        assertThat(result.getWarnings()).containsOnly("warning");
-        assertThat(result.getErrors()).containsOnly("error");
+        verify(exceptionRecordFinalizer, never()).finalizeExceptionRecord(anyMap(), anyLong());
     }
 
     @Test
-    void should_return_service_case_when_it_already_exists_in_ccd_for_a_given_exception_record() throws Exception {
+    void should_return_existing_case_if_it_exists_in_ccd_for_a_given_exception_record() throws Exception {
         // given
         setUpTransformationUrl();
 
         when(ccdApi.getCaseRefsByBulkScanCaseReference(Long.toString(CASE_ID), null))
-            .thenReturn(singletonList(345L));
+            .thenReturn(asList(345L));
+        Map<String, Object> caseData = basicCaseData();
 
         // when
         ProcessResult result = service.process(new CcdCallbackRequest(
             EVENT_ID_CREATE_NEW_CASE,
-            caseDetails(basicCaseData()),
+            caseDetails(caseData),
             true
         ), IDAM_TOKEN, USER_ID);
 
         // then
-        assertThat(result.getExceptionRecordData().get(CASE_REFERENCE)).isEqualTo("345");
-        assertThat(result.getWarnings().isEmpty()).isTrue();
-        assertThat(result.getErrors().isEmpty()).isTrue();
+        assertThat(result.getExceptionRecordData()).isEmpty();
```

Comment:
Worth setting up the finaliser to return some simple non-empty map and this assertion would then verify it's the same map.

Reply:
Have done

Reply:
where?